client/src/utils/store.js

```diff
@@ -186,11 +186,12 @@ const StoreProvider = ({ children }) => {
     dispatch({ type: "LOADING" });
     let file = zipFile;
     let uzip = await Zip.unzip(file);
+    let date = file.lastModifiedDate || file.lastModified;
```

Comment:
When do we see the second case? Is this based on the new twitter archive file?

Reply:
This is a Safari vs Chrome interface difference